astropy/coordinates/sky_coordinate.py

```diff
@@ -1225,6 +1225,92 @@ def from_pixel(cls, xp, yp, wcs, origin=0, mode='all'):
         """
         return pixel_to_skycoord(xp, yp, wcs=wcs, origin=origin, mode=mode, cls=cls)
 
+    def radial_velocity_correction(self, kind='barycentric', obstime=None,
+                                   location=None):
+        """
+        Compute the correction required to convert a radial velocity at a given
+        time and place to a barycentric or heliocentric velocity.
+
+        Parameters
+        ----------
+        kind : str
```

Comment:
No.  You're the one who always wants long names! 😉 

Seriously, though, I agree but couldn't come up with anything better either, other than `bary=True`/`bary=False` which I dislike even more